src/views/add-team-member.html

```diff
@@ -19,12 +19,17 @@ <h1 class="govuk-heading-l">Add a GitHub member to a team</h1>
 
         {% include "include/inputs/team-name.html" %}
 
-        {% set githubHandleText = "Member GitHub handle" %}
-        {% include "include/inputs/github-handle.html" %}
+        {% set githubHandlesText = "Member GitHub handle(s)" %}
+        {% set githubHandlesHint = "For multiple members Enter GitHub handles separated by commas. For example: username1,username2,username3. " %}
```

Comment:
Re-phrase (space separate examples as it's easier to read): For multiple members, enter GitHub handles as comma-separated values. For example, username1, username2, username3